tests/foreman/api/test_capsulecontent.py

```diff
@@ -777,14 +777,11 @@ def test_positive_capsule_pub_url_accessible(self, module_capsule_configured):
 
         :BZ: 1463810, 2122780
         """
-        https_pub_url = f'https://{module_capsule_configured.ip_addr}/pub'
-        http_pub_url = f'http://{module_capsule_configured.ip_addr}/pub'
+        https_pub_url = f'https://{module_capsule_configured.hostname}/pub'
+        http_pub_url = f'http://{module_capsule_configured.hostname}/pub'
         for url in [http_pub_url, https_pub_url]:
             response = client.get(url, verify=False)
```

Comment:
Now that the hostname is used, you could (provided certificates are set up correctly) also verify the response. While not part of this PR, I think it would be nice

Reply:
To be explicit, I'd like to drop `verify=False` (since verification is on by default). Probably not in this PR, so consider it more of a general remark.

Reply:
When the verification is on the "https" fails due to self-signed cert on the capsule
```
>           self._sslobj.do_handshake()
E           ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate in certificate chain (_ssl.c:1006)
```

Reply:
Right, so please see it as future work. My idea is that in the real world users are going to validate it so our test suite should do the same. Ideally the `client` would be set up with the proper certificates to validate we have also set up the HTTPS chain correctly. We could have bugs in there and from a platform perspective I'd like it if robotello verifies that for us.

Reply:
Ah, gotcha. Here the (nailgun) `client` is just a wrapper for `requests`, so it would probably need to handle different certs (for different SATs and CAPSes). Or maybe just one common signing authority?
Yeah, that is a bit out of scope of this PR but definitely interesting to think about.

Reply:
You can (should) load the CA and that's the same between sat and capsule

Reply:
+1 to what @ekohl suggested, it would be nice to have this later on at some point to simulate what real users are doing.